modules/ROOT/pages/reference/procedures.adoc

```diff
@@ -698,6 +713,51 @@ Replaced by: `ALTER USER`.
 
 == Procedure descriptions
 
+[[procedure_cdc_current]]
+.cdc.current()
+[cols="<15s,<85"]
+|===
+| Description
+a|
+Fetch the current `changeId` for use with xref:reference/procedures.adoc#procedure_cdc_unfiltered_query[`cdc.unfiltered.query()`]
+
+For mor details, see link:{neo4j-docs-base-uri}/change-data-capture/{page-version}/introduction[Change Data Capture].
+| Signature
+m|cdc.current() :: STRING NOT NULL
```

Comment:
Is this what is returned from `SHOW PROCEDURES`? I don't think we can currently say it will be `NOT NULL` in the signatures in the database so I don't think it'll be here either (even if null never gets returned), this would be for the same reason as everything previously having `?` regardless of if null could be sent in/returned or not

Reply:
also applies to the other two procedures

Reply:
the docs is no longer autogenerated so you have to write them by hand here, but it should match the signature from SHOW PROCEDURES

Reply:
Gem just merged her [PR](https://github.com/neo-technology/neo4j/pull/21710/files#diff-67dfcef491128b64eef569dc1b5ef578c493ff9f6cc79fb278846f0975d855e8) that updates the signatures to the new Cypher types and I assume she'll update the existing docs soon to match. So if this PR will be merged after those updates we need to remember to update the signatures before merge

Reply:
Or we could merge first. This PR no longer points to the unpublished docs, so I think we are ok to merge? @renetapopova

Reply:
hence my `if` XD (just make sure Gem's aware of you adding new procedures that'll need updating as well so they don't get missed if you merge first)